docs/prepare-db-upgrade.md

```diff
@@ -69,15 +69,15 @@ Although we have some automated testing of the scripts (e.g. to test that you ca
 To test the upgrade script, run:
 
 ```
-codeql test run --search-path=<old-extractor-pack>;ql <test-dir>
+codeql test run --search-path=<old-extractor-pack> --search-path=ql <test-dir>
 ```
 
 Where `<old-extractor-pack>` is an extractor pack containing the old extractor and dbscheme that pre-date your changes, and `<test-dir>` is the directory containing the qltests for your language. This will run the tests using an old extractor, and the test databases will all be upgraded in place using your new upgrade script.
 
 To test the downgrade script, create an extractor pack that includes your new dbscheme and extractor changes. Then checkout the `main` branch of `codeql` (i.e. a branch that does not include your changes), and run:
 
 ```
-codeql test run --search-path=<new-extractor-pack>;ql <test-dir>
+codeql test run --search-path=<new-extractor-pack> --search-path=ql <test-dir>
```

Comment:
I couldn't get this command to run somehow. Does it work for you?

Reply:
Yes, it works for me, although it appears the `--search-path=ql` is not needed in this case.

Reply:
ok, not sure what I was doing wrong. I did expect that `--search-path=ql` wasn't needed, but failed to validate that. Thanks for confirming. Shall we remove it ?